modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-adding-templates-to-custom-namespace.adoc
+
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+The `ssp-operator` deploys Red Hat templates to the `openshift` namespace by default. Templates in the `openshift` namespace are publicly availably to all users. To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.
```

Comment:
> To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.

Setting a custom namespace will allow for the modification or deletion of VM templates in the openshift namespace